FullJournalImportExportPlugin.inc.php

```diff
@@ -332,9 +332,7 @@ function tarFiles($targetPath, $targetFile, $sourceFiles) {
 
 	function _untarFile($targetPath, $targetFile) {
 		assert($this->_checkedForTar);
-		$tarCommand = Config::getVar('cli', 'tar') . ' -xzf ' . escapeshellarg($targetFile);
-		$tarCommand .= ' -C ' . escapeshellarg($targetPath);
-		exec($tarCommand);
+		exec(Config::getVar('cli', 'tar') . ' -xf ' . escapeshellarg($targetFile) . ' -C ' . escapeshellarg($targetPath));
```

Comment:
Guess i'll have to use the switch case again then.
Later today I'm gonna send the fix.